src/zenml/new/steps/log_artifact_metadata.py

```diff
@@ -0,0 +1,120 @@
+#  Copyright (c) ZenML GmbH 2023. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+"""Utility functions for logging artifact metadata."""
+
+from typing import Dict, Optional
+
+from zenml.exceptions import StepContextError
+from zenml.metadata.metadata_types import MetadataType
+from zenml.new.steps.step_context import get_step_context
+
+
+def log_artifact_metadata(
+    output_name: Optional[str] = None,
+    description: Optional[str] = None,  # TODO: Do we really need this?
```

Comment:
I'd suggest to remove this arg here actually to keep the function maximally general, WDYT?

Reply:
Removed it 👍